vision/obstacle/obstacle_tracker.py

```diff
@@ -0,0 +1,95 @@
+"""
+Keeps track of detected obstacles between frames
+"""
+
+import cv2
+import numpy as np
+from vision.bounding_box import BoundingBox
+
+
+class Obstacle:
+    """
+    Properties of an obstacle in the frame.
+
+    Parameters
+    ----------
+    bbox: BoundingBox
+        bounding box of obstacle 
+    """
+    def __init__(self, bbox: BoundingBox):
+        self.bounding_box = bbox
+        self.center = np.mean(bbox)

```

Comment:
`np.mean(bbox)` will not work because bbox is a user created class. You need to take the mean of bbox.vertices
Also, we want to separate the x and y values, so we should take the mean along the 0th axis.

`np.mean(bbox.vertices, axis=0)` will return [xval, yval] with a datatype of ndarray and datatypes of x and y as floats.